Broken MonoKickstart compatibility since 3.5.1

PR #4534 broke MonoKickstart compatibility which is now throwing this error, on both OS X and Linux:
> Fontconfig error: Cannot load config file

It looks like the method used to test libgdiplus triggers a font search in the Mono font cache, which doesn't exist with MonoKickstart (and only exist if the Mono SDK is installed, which is why it doesn't fail on a dev environment).

This most probably also affects projects packaged with Xamarin.

@hach-que @cra0zy

I am having some trouble right now with trying to link mono statically:
```
/usr/bin/ld: cannot find -lmono-2.0
collect2: error: ld returned 1 exit status
ERROR: [Fail] 256
```

Anyway, will post a video tomorrow once I figure out what's wrong.

--- a/MonoGame.Framework/Desktop/OpenTKGameWindow.cs
+++ b/MonoGame.Framework/Desktop/OpenTKGameWindow.cs
@@ -390,26 +390,6 @@ namespace Microsoft.Xna.Framework
             // try to load it using Mono, it will cause a crash because of this.
             try
             {
-                // If we're running on a system where libgdiplus is not available, we must
-                // force an attempted load here.  When "new Icon" runs and fails here because
-                // of the failure to load the library, the object is still allocated.  Later on
-                // when the GC attempts to clean up the Icon object, that will attempt to load
-                // the library in it's finalizer and throw an exception.  As per the .NET documentation
-                // when a finalizer throws an exception, the application is immediately terminated
-                // with no recovery.
-                if (t != null)
-                {
-                    var drawingTypes = typeof(Icon).Assembly.GetTypes();
-                    foreach (var type in drawingTypes)
-                    {
-                        if (type.Name == "GDIPlus")
-                        {
-                            type.GetMethod("RunningOnUnix").Invoke(null, null);
-                            break;
-                        }
-                    }
-                }
-
                 if (t == null && assembly != null)
                     window.Icon = Icon.ExtractAssociatedIcon(assembly.Location);
                 else {

